### PR TITLE
Support more commands, reorder, raise error with screenshot and camshot

### PR DESCRIPTION
**Pull request by infeeeee**
Thank you for this package.

Added some missing commands.

Commands used to be in the order of contributions. I reordered them similar to official docs, and grouped them by similarity. Much easier to read the code this way.

`getCamshot` and  `getScreenshot` now raises an error just like any other commands.

Tested this with my tablet running "Fully Single App Kiosk" .

Supersedes #18

### CODE REVIEW

File: fullykiosk/__init__.py [hunk 1]
```diff
@@ -47,91 +52,136 @@ def deviceInfo(self):
...
 
-    async def restartApp(self):
-        await self.sendCommand("restartApp")
+    async def stopScreensaver(self):
+        await self.sendCommand("stopScreensaver")
 
-    async def loadStartUrl(self):
-        await self.sendCommand("loadStartUrl")
+    async def startDaydream(self):
+        await self.sendCommand("startScreensaver")
```
Comment (cgarwood) [inline fullykiosk/__init__.py]: Shouldn't this send the `startDaydream` command instead of `startScreensaver`?
Same for stopDaydream below

**Review by cgarwood — CHANGES_REQUESTED** [review]
Thanks! One comment on the start/stop daydream commands, but otherwise looks good.

File: fullykiosk/__init__.py (hunk 1 above)
Comment (infeeeee) [reply fullykiosk/__init__.py]: Yes, I copypasted the lines, and forgot to change that part as well :)

**Review by cgarwood — APPROVED** [review]
Thanks!

**Comment by neubi4** [comment]
Hi @cgarwood 

is it possible to release a new version of the library? I would like to use the new triggerMotion command from this PR,, but there is no new release after this merge.

**Pull request merged**